fastify.js

```diff
@@ -348,6 +348,15 @@ function fastify (options) {
   }
 
   Object.defineProperties(fastify, {
+    listeningOrigin: {
+      get () {
+        const address = this.addresses().slice(-1).pop()
+        if (typeof address === 'string') {
```

Comment:
I believe we will need to ignore this line to avoid the coverage issue in Windows, as this is not testable for the os platform.